Swift/BRCrypto/BRCryptoSystem.swift

```diff
@@ -76,30 +182,44 @@ public final class System {
     /// - Returns: The default AddressScheme
     ///
     public func defaultAddressScheme (network: Network) -> AddressScheme {
-        switch network.currency.code {
-        case Currency.codeAsBTC: return AddressScheme (core: CRYPTO_ADDRESS_SCHEME_BTC_SEGWIT)
-        case Currency.codeAsBCH: return AddressScheme (core: CRYPTO_ADDRESS_SCHEME_BTC_LEGACY)
-        case Currency.codeAsETH: return AddressScheme (core: CRYPTO_ADDRESS_SCHEME_ETH_DEFAULT)
-        default: return AddressScheme (core: CRYPTO_ADDRESS_SCHEME_GEN_DEFAULT)
-        }
+        return defaultAddressSchemeMap[network.uids] ?? .genDefault
     }
 
     ///
+    /// Wallet Manager Modes
```

Comment:
This should really be at the C level, but time is a factor.

Reply:
agreed.